.github/workflows/ci.yml

```diff
@@ -60,6 +60,7 @@ jobs:
     - run: yarn test-legacy-javascript
     - run: yarn i18n:checks
     - run: yarn dogfood-lhci
+    - run: yarn generate-insight-audits
```

Comment:
Why does this need to be in our CI? Is it to catch when there's a new insight in the trace engine that we don't have an implementation for?

Reply:
Ope just read the PR description. SGTM.